code/modules/surgery/organs/internal/cyberimp/augments_arms.dm

```diff
@@ -534,3 +543,21 @@
 	COOLDOWN_START(src, slam_cooldown, slam_cooldown_duration)
 
 	return COMPONENT_CANCEL_ATTACK_CHAIN
+
+/datum/status_effect/organ_set_bonus/strongarm
+	id = "organ_set_bonus_strongarm"
+	organs_needed = 2
+	bonus_activate_text = span_notice("Your improved arms allow you to open airlocks by force with your bare hands!")
+	bonus_deactivate_text = span_notice("You can no longer force open airlocks with your bare hands.")
+
+/datum/status_effect/organ_set_bonus/strongarm/enable_bonus()
+	. = ..()
+	if(!.)
+		return
+	owner.AddElement(/datum/element/door_pryer, pry_time = 6 SECONDS, interaction_key = DOAFTER_SOURCE_STRONGARM_INTERACTION)
+
+/datum/status_effect/organ_set_bonus/strongarm/disable_bonus()
+	. = ..()
+	owner.RemoveElement(/datum/element/door_pryer)
```

Comment:
```suggestion
owner.RemoveElement(/datum/element/door_pryer, pry_time = 6 SECONDS, interaction_key = DOAFTER_SOURCE_STRONGARM_INTERACTION)
```

People oft forget that removing bespoke elements requires using the exact same arguments of when it was added.